File: 00_Learn_The_Basics/Things_To_Know_In_A_Language/While_loops/Reverse.java
//Finding the reverse of the number entered by the user

import java.util.*;

public class Reverse
{
    
    public static void main(String args[])
    {
        int r=0;
        Scanner in=new Scanner(System.in);
        int N;
        System.out.println("Enter the number");
        N=in.nextInt();
        while(N>0)
        {
            r=r*10+(N%10);
            N=N/10;
        }

        System.out.println("The reverse of the number  is : " + r);

    }
}

File: 00_Learn_The_Basics/Things_To_Know_In_A_Language/While_loops/reverse.cpp
//Finding the reverse of the number entered by the user

#include<iostream>
using namespace std;

int main()
{
        int r=0,N;
        cout<<"Enter the number"<<endl;
        cin>>N;
        while(N>0)
        {
            r=r*10+(N%10);
            N=N/10;
        }

        cout<<"The reverse of the number  is : " << r;

}
